src/core/server/base.js

```diff
@@ -302,7 +297,7 @@ function baseServer(
         let sagas = appSagas;
         if (!sagas) {
           // eslint-disable-next-line global-require, import/no-dynamic-require
-          sagas = require(`${appName}/sagas`).default;
+          sagas = require(`${APP_NAME}/sagas`).default;
```

Comment:
I suppose we could also use `amo` here as well, given that we are requiring a unique file.